### DISCUSSION TITLE
Headless Inferencer cannot generate the User Interface

### DISCUSSION THREAD

**Discussion by tegarmonster** [refinedev/refine#5556] [Q&A]
Hello, I'm using refine and NextJs..
I want to generate UI using Inferencer, my data source come from internal API nextJs (API Routes)
but seem not work, whats wrong with my code?

![Screenshot 2024-01-24 150945](https://github.com/refinedev/refine/assets/137743780/4b3b905f-3c85-4331-938d-22b284ab10a9)
![Screenshot 2024-01-24 151129](https://github.com/refinedev/refine/assets/137743780/6f57841a-47c8-481d-bf29-3af02a125139)

**Comment by BatuhanW** [comment]
Hello @tegarmonster you can ask questions on our discord channel, try to provide as much as details possible, so our bot can help.

https://discord.com/channels/837692625737613362/1019855665633706044
